fern/docs.yml

```diff
@@ -354,6 +354,9 @@ navigation:
           - page: Publish your Docs
             path: ./pages/docs/getting-started/publishing-your-docs.mdx
             icon: fa-regular fa-upload
+          - page: Customer Showcase
```

Comment:
why put it this low? isn't this one of the first things you'd want someone evaluating Fern to see?

could see Getting Started just being:
- Overview
- Showcase
- Quickstart

don't have clear guidance on where pages like Global Config and Project Structure would go though